packages/5schatten/emulation/standalone/ppsspp/package.mk

```diff
@@ -29,7 +29,7 @@ configure_package() {
 
   # OpenGLES Support
   if [ "${OPENGLES_SUPPORT}" = "yes" ]; then
-    PKG_DEPENDS_TARGET+=" ${OPENGLES}"
+    PKG_DEPENDS_TARGET+=" ${OPENGLES} libX11"
```

Comment:
Could you add another `if` statement which only adds libX11 if the device is RPi4? For Mali based builds libX11 is not needed.

Reply:
When I reviewed the cMakeLists.txt I overlooked that the code I reviewed needing x11 is only for GLES2. I have made the change.